### DISCUSSION TITLE
24.10 orange pi 5 , How to boot from sata?

### DISCUSSION THREAD

**Discussion by hakotani-o** [Joshua-Riek/ubuntu-rockchip#1106] [Q&A]
I can not find overlay for orangepi-5-sata.dtbo.

Thank you.

**Comment by Bahazzie** [comment]
It's possible. Even on this forum are good descriptions. I have a SATA M2 mini SSD and it boots fine. Do a bit of searching here.

**Reply by hakotani-o** [reply]
Thank you for your reply.

The other day, I tried the 6.12-rc5 kernel and it booted, 
but the next day I couldn't boot Opi5. 
In MASKROM mode, Opi5 was restored by writing MiniLoaderAll.bin.

At the same time,
 I connected the JMB582 board and SSD Adapter to Orange pi 5 and changed it to be able to boot from 2.5inch SSD.
This change allows you to use SATA BOOT without worrying.

thank you.

**Comment by ageevp** [comment]
Known workaround.

Clone SD installation to USB device.
Change UUID of USB device by gparted.
Edit SD /boot/extlinux/extlinux.conf.
Replace SD UUID by USB UUID.
